NEWS

```diff
@@ -1,4 +1,11 @@
 2.17.1 (2018-mm-dd)
```

Comment:
We should set the proper date here :)

Reply:
Yep, at release time! I don't want to set a date and then delay the release for any reason.